ui/ozone/platform/wayland/wayland_keyboard.cc

```diff
@@ -63,8 +63,8 @@ void WaylandKeyboard::Keymap(void* data,
   bool success =
       KeyboardLayoutEngineManager::GetKeyboardLayoutEngine()
           ->SetCurrentLayoutFromBuffer(keymap_str, strnlen(keymap_str, size));
-  DCHECK(success) << "Failed to set the XKB keyboard mapping.";
-  munmap(keymap_str, size);
+  if (success)
+    munmap(keymap_str, size);
```

Comment:
sorry, forgot about this.

I wonder if would could do something like

...
````
if (!success) {
  LOG(ERROR) << "Failed to set the XKB keyboard mapping. Only basic keyboard support available.";
  return;
}

munmap(keymap_str, size);
}

Reply:
Done